tests/osaka/eip7692_eof_v1/eip4750_functions/test_callf_execution.py

```diff
@@ -663,3 +669,53 @@ def test_callf_max_stack(
         sender=sender,
     )
     state_test(env=env, pre=pre, post=post, tx=tx)
+
+
+def test_callf_retf_memory_context(
+    state_test: StateTestFiller,
+    pre: Alloc,
+):
+    """Verifies CALLF and RETF don't corrupt memory."""
+    env = Environment()
+    contract_address = pre.deploy_contract(
+        code=Container(
+            sections=[
+                Section.Code(
+                    Op.SSTORE(slot_code_worked, value_code_worked)
+                    + Op.MSTORE(0, 1)
+                    + Op.CALLF[1]
+                    + Op.SSTORE(slots_extra[0], Op.MSIZE())
```

Comment:
We could replace the slot approach with a `Storage` instance.

```
storage = Storage()
```
then you can use `store_next` with the expected value every time you need to store something:
```
Op.SSTORE(storage.store_next(VALUE_YOU_EXPECT), Op.MSIZE)
```

At the end you simply pass the storage object in the account in the post.